vllm/model_executor/models/ultravox.py

```diff
@@ -231,6 +231,7 @@ def forward(self, audio_embeds: torch.Tensor) -> torch.Tensor:
 
 class FlippedSiluAndMul(SiluAndMul):
     """Ultravox is trained with SwiGLU with flipped halves."""
+    name = "flipped_silu_and_mul"
```

Comment:
Removed

Reply:
Sorry, to clarify: this is a subclass of another CustomOp and it's in a model definition so I figured it didn't need to be registered. It can be enabled/disabled using the parent name (`"silu_and_mul"`), but let me know if you think we should register it